data_workspace: acceptance test regression

**Acceptance error**
> === RUN   TestAccDataSourceWorkspace_basic
    data_workspace_test.go:14: Step 1/1 error: Check failed: Check 4/4 error: data.bitbucket_workspace.test: Attribute 'is_private' expected "true", got "false"
--- FAIL: TestAccDataSourceWorkspace_basic (1.39s)

--- a/bitbucket/data_workspace_test.go
+++ b/bitbucket/data_workspace_test.go
@@ -21,7 +21,7 @@ func TestAccDataSourceWorkspace_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(dataSourceName, "workspace", workspace),
 					resource.TestCheckResourceAttrSet(dataSourceName, "name"),
 					resource.TestCheckResourceAttrSet(dataSourceName, "slug"),
-					resource.TestCheckResourceAttr(dataSourceName, "is_private", "true"),
+					resource.TestCheckResourceAttrSet(dataSourceName, "is_private"),
 				),
 			},
 		},

